doc/source/example_notebooks/IS2_data_access2-subsetting.ipynb

```diff
@@ -114,10 +106,8 @@
     "user_expressions": []
    },
    "source": [
-    "By default, spatial and temporal subsetting based on your initial inputs is applied to your order unless you specify `subset=False` to `order_granules()` or `download_granules()` (which calls `order_granules` under the hood if you have not already placed your order) functions.\n",
-    "Additional subsetting options must be specified as keyword arguments to the order/download functions.\n",
-    "\n",
-    "Although some file format conversions and reprojections are possible using the `format`, `projection`,and `projection_parameters` keywords, the rest of this tutorial will focus on variable subsetting, which is provided with the `Coverage` keyword."
+    "By default, spatial and temporal subsetting based on your initial inputs is applied to your order unless you specify `subset=False` to the `order_granules()` function or the `download_granules()` function, which calls `order_granules` under the hood if you have not already placed your order.\n",
```

Comment:
```suggestion
    "By default, spatial and temporal subsetting based on your initial inputs is applied to your order. This will be true no matter if you  use the `order_granules()` function or the `download_granules()` function (which calls `.order_granules under()` the hood if you have not already placed your order). If you don't want your order to be spatially subset, you can use the `subset=False` argument in either `.order_granules()` or `.download_granules()`.\n",
```